### DISCUSSION TITLE
export config

### DISCUSSION THREAD

**Discussion by gapochat** [Gozargah/Marzban#355] [General]
I configured some cinfigs by other panels and want to have them on Marzban or
I have some cinfigs on v2raNG and want to export them to put in Marzban setting and use them.

من میخوام همون کانفیگی که دستی با سایر پنل ها میسازم رو داخل مرزبان داشته باشم. چجوری باید این کارو انجام بدم؟ میدونم باید فایل setting رو ادیت کنم و اضافه کنم ولی کد موردنظرمو ندارم و میخوام که عینا همون کانفیگ بیاد مثلا grpc روی multi قرار بگیره و...
اینجا هم نبود : https://github.com/XTLS/Xray-examples

**Comment by amirspyware** [comment]
export config 
با v2rayn 
نسخه سرور
